recipes/sfml/all/conanfile.py

```diff
@@ -91,10 +107,12 @@ def generate(self):
         tc.variables["SFML_BUILD_AUDIO"] = self.options.audio
         tc.variables["SFML_INSTALL_PKGCONFIG_FILES"] = False
         tc.variables["SFML_GENERATE_PDB"] = False
-        tc.variables["SFML_USE_SYSTEM_DEPS"] = True
+        tc.variables["SFML_USE_SYSTEM_DEPS"] = Version(self.version) < "3.0.0"
```

Comment:
Why change this? A package manager probably wants to use its own dependencies instead of the dependencies bundled with the library.

Reply:
It was a pain getting things to work without. One ends up having to patch how things are searched for.

Apparently, in more recent versions of conan, it's better at building them. This resolved all of the package finding issues. I imagine this will be a more consistent build as a result, but that's just conjecture on my part.